apps/wallet-mobile/src/components/Button/Button.tsx

```diff
@@ -138,7 +138,7 @@ const useStyles = () => {
       borderColor: color.primary_c600,
     },
     text: {
-      color: color.gray_cmin,
+      color: isDark ? color.white_static : color.gray_cmin,
```

Comment:
@banklesss can you check with Dima, it shouldn't have this type of check in the theme, it should use a `sub-token` -> `color.title-1`